lib/capybara/spec/session/has_text_spec.rb

```diff
@@ -39,6 +39,19 @@
     expect(@session).to have_text('text with whitespace', normalize_ws: true)
   end
 
+  context 'with enabled default collapsing whitespace' do
+    before do
+      @previous_default_normalize_ws = Capybara.default_normalize_ws
```

Comment:
Since normalize_ws is reset in spec_helper there's no need to store/restore the previous